templates/spin_sidebar.hbs

```diff
@@ -14,6 +14,7 @@
         </p>
         <ul class="menu-list">
             <li><a {{#if (active_content request.spin-path-info "/spin/writing-apps" )}} class="active" {{/if}} href="{{site.info.base_url}}/spin/writing-apps">Writing Applications</a></li>
+            <li><a {{#if (active_content request.spin-path-info "/spin/designing-spin-apps" )}} class="active" {{/if}} href="{{site.info.base_url}}/spin/designing-spin-apps">Designing Applications</a></li>
```

Comment:
Suggestion: move this below "Running Applications" so that the TOC fully guides people through the basics of `new > build > up` before guiding them into a deeper dive on application design.

Reply:
I have updated the order here as suggested. Thanks.